vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -359,6 +426,10 @@ void LedgerImpl::OnLedgerStateLoaded(
 
 void LedgerImpl::SetConfirmationsWalletInfo(
     const ledger::WalletInfoProperties& wallet_info_properties) {
+  if (!bat_confirmations_) {
```

Comment:
nit: should we use `IsConfirmationsRunning` for this ones as well?

Reply:
Fixed